src/bluecherry.c

```diff
@@ -439,6 +445,85 @@ static int _bluecherry_dtls_recv(void *ctx, unsigned char *buf, size_t len)
   return recv(sock, buf, len, 0);
 }
 
+static esp_err_t bluecherry_connect(void)
+{
+    int ret;
+    const int handshake_total_timeout_s = 10; /* tune as needed */
+
+    ESP_LOGI(TAG, "Connecting to the BlueCherry platform");
```

Comment:
I like to keep the library clean of LOGI/D and only include LOGE. This to keep the footprint as small as possible. It's up to the end user to add extra logs in the application layer.